### ISSUE TITLE
Track the last focused chat message so that moving between chat input and response list is smoother for kb users

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#268238]
Idea from @rperez030 : Provide a mechanism to move between the input field and the chat thread without losing focus. Right now, Ctrl+Up Arrow always moves focus to the latest message in the thread. We would need a mechanism to return to the chat that moves focus not to the latest message, but to the last active message, knowing that not all Windows screen readers are good at respecting focus events in a web view.

**Comment by meganrogge** [comment]
My feeling is we should have a separate keybinding for the sake of muscle memory.

**Comment by meganrogge** [comment]
I think just adding `Shift` here would work well

**Comment by rperez030** [comment]
I agree. Either that or a new setting. What about adding the chat list to the **F6** cycle? Right now, pressing **F6** from the input field moves focus to the status bar. If it went to the selected message in the list before reaching the status bar, it would be fully aligned with the request in [#267152](https://github.com/microsoft/vscode/issues/267152).

#### Current Behavior

While in the input field, **F6** moves focus to the status bar, and **Shift+F6** moves focus back to the chat.

#### New Behavior

While in the input field, **F6** moves focus to the selected message in the chat thread, then to the status bar. **Shift+F6** then moves focus back to the selected message, then to the input field.

**Comment by meganrogge** [comment]
Related https://github.com/microsoft/vscode/issues/267152

**Comment by meganrogge** [comment]
Looking into this, it's actually not related to tab index. Currently, when I use `f6`, the panel chat's input box is focused, which I think seems good. @rperez030 are you suggesting instead it should focus the most recently focused part of the panel?

**Comment by rperez030** [comment]
> ... [@rperez030](https://github.com/rperez030) are you suggesting instead it should focus the most recently focused part of the panel?

Not exactly. The new command to move to the last focused message is a great improvement, and I'm sure I'm going to **start** using it by default immediately. However, it is still a key binding to discover, learn, and memorize, and I know users who don't even know they can jump to the list with `Cmd + Up Arrow`.

One of the pain points mentioned in [#267152](https://github.com/microsoft/vscode/issues/267152) is the inability to use `F6` and `Shift + F6` to navigate between the chat and the input box. `F6`, like the `Tab` key, are well-known ways to move between sections for keyboard users. 

Maybe I can demonstrate this with a video.
